poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFWorkbook.java

```diff
@@ -786,15 +798,6 @@ public void remove() throws IllegalStateException {
         }
     }
 
-    /**
-     * Alias for {@link #sheetIterator()} to allow
-     * foreach loops
-     */
-    @Override
-    public Iterator<Sheet> iterator() {
```

Comment:
add this back

Reply:
I've moved this to the `Workbook` parent interface, so its essentially the same (alias for `sheetIterator()`), see https://github.com/apache/poi/pull/290/files#diff-a4d997b0b0814dae7bbc74ef62d8b413d102a7eae2b40aa5bf1ce4a7840a9233